app/views/accounts/personal_keys/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<% title(t('account.personal_key.get_new')) %>
+
+<h1><%= t('account.personal_key.get_new') %></h1>
+
+<p class="margin-top-4">
+  <%= t('account.personal_key.get_new_description') %>
+</p>
+
+<%= button_to(
+      create_new_personal_key_path,
+      method: 'post',
+      class: 'usa-button usa-button--wide usa-button--big margin-top-4') do %>
+  <%= t('forms.buttons.continue') %>
+<% end %>
+
+
+<div class="margin-top-4 border-top border-primary-light">
+  <div class="margin-top-1">
+    <%= link_to(t('links.cancel'), account_path) %>
+  </div>
+</div>
```

Comment:
Thoughts on using the shared partial here? Might need to adjust button margin to keep the same effective margins.

```suggestion
<%= render 'shared/cancel', link: account_path %>
```

Reply:
Totally forgot we had this partial, I just copied this wholesale from the "forget all browsers" page --- thanks! https://github.com/18F/identity-idp/blob/63504b9d418fdebe0da538ced772d4bc9cbfaf24/app/views/users/forget_all_browsers/show.html.erb#L13-L17

I'll make a note to refactor that one too

Reply:
Created a ticket to follow up on refactoring elsewhere: https://cm-jira.usa.gov/browse/LG-5705